packages/sku/config/storybook/config.js

```diff
@@ -1,13 +1,41 @@
-const fs = require('fs');
-const path = require('path');
-const { paths, storybookAddons } = require('../../context');
+import fs from 'fs';
+import path from 'path';
+import { paths, storybookAddons, storybookStoryStore } from '../../context';
 
-module.exports = {
+/** @type {import("@storybook/react-webpack5").StorybookConfig} */
+export default {
   stories: paths.src
     .filter((srcPath) => fs.statSync(srcPath).isDirectory())
     .map((srcPath) => path.join(srcPath, '**/*.stories.@(js|ts|tsx)')),
   addons: storybookAddons,
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {},
+  },
   core: {
-    builder: 'webpack5',
+    builder: {
+      name: '@storybook/builder-webpack5',
+      options: {
+        fsCache: true,
+      },
+    },
+  },
+  features: {
+    storyStoryV7: storybookStoryStore,
   },
+  babel: (config) => ({
+    ...config,
+    presets: [
+      ...config.presets,
+      [
+        '@babel/preset-env',
```

Comment:
Do we need to `require.resolve` these? Would it still work with strict/default pnpm (no `shamefully-hoist`)?

Reply:
Hmm, not sure honestly, depends how babel resolves them. I suppose it couldn't hurt to `require.resolve` them, just in case.